src/enzo/Make.mach.tigercpu

```diff
@@ -0,0 +1,146 @@
+#=======================================================================
+#
+# FILE:        Make.mach.tigercpu
+#
+# DESCRIPTION: Makefile settings for tigercpu at Princeton
+#
+# AUTHOR:      He Jia (hejia@princeton.edu)
+#
+# DATE:        2020-08-31
+#
+#=======================================================================
+
+MACH_FILE  = Make.mach.tigercpu
+MACH_TEXT  = tigercpu
+MACH_VALID = 1
+
+#       please load these modules:
+#               module load intel/19.1/64/19.1.1.217 openmpi/intel-19.0/3.1.3/64 hdf5/intel-16.0/1.8.16 anaconda/2019.10
+#       then in a conda env, please do:
+#               conda install yt mercurial -c conda-forge
+#               conda install python-hglib -c mforbes 
+#       building:
+#               ./configure 
+#               cd src/enzo
+#               make machine-tigercpu
+#               gmake opt-high papi-no #grackle-yes
+#               make show-config
+#               make
+#               cd ../inits/
+#               make
+#               cd ../ring/
+#               make
+
+#-----------------------------------------------------------------------
+# Install paths (local variables)
+#-----------------------------------------------------------------------
+
+LOCAL_COMPILER = ${PROD_DIR}/compiler
+LOCAL_MPI_INSTALL = ${MPI_HOME}
+LOCAL_HDF5_INSTALL = ${HDF5DIR}
```

Comment:
If any combination of these are defined by the user (e.g. in the `.bashrc`), and not when a module is loaded, I would suggest putting the full path here, substituting system variables like `$USER` as appropriate.

Reply:
Hi, I believe those paths are defined when the corresponding modules are loaded.

Reply:
Excellent! I'll remove the change request and approve.

Reply:
Why all the mercurial? Neither enzo nor yt use anymore. (And yt should not be required for installing enzo.)